frameworks/portworx/tests/test_sanity.py

```
@@ -45,6 +45,6 @@ def test_uninstall_package():
 # Do post uninstall cleanup
 @pytest.mark.sanity
 def test_post_uninstall_cleanup():
-    assert sdk_install.portworx_cleanup() == 0
-
-
+    if sdk_install.portworx_cleanup() != 0:
```

What I meant was that portworx_cleanup() method should raise an exception on failure instead of returning a code. Here the code will be just calling `sdk_install.portworx_cleanup()`